nano/node/blockprocessor.hpp

```diff
@@ -88,6 +89,7 @@ class block_processor final
 	nano::write_database_queue & write_database_queue;
 	nano::mutex mutex{ mutex_identifier (mutexes::block_processor) };
 	nano::state_block_signature_verification state_block_signature_verification;
+	std::thread thread;
```

Comment:
Maybe `processing_thread`? Or any other name just to actually say something about the purpose.

Reply:
Way better than just "thread".